tools/zbreplay

```diff
@@ -49,8 +49,12 @@ parser.add_argument('-s', '--sleep', action='store', type=float, default=1.0)
 parser.add_argument('-D', action='store_true', dest='showdev')
 args = parser.parse_args()
 
-if args.showdev:
-    show_dev()
+if args.showdev and not args.devstring:
```

Comment:
See above note for this and other tools

Reply:
Same. Auto-Discovery does not work for me.